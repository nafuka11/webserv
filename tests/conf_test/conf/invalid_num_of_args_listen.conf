server {
    listen 4242 4200;
}