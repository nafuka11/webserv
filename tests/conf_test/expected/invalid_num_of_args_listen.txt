webserv: invalid number of arguments in "listen" directive in ./conf/invalid_num_of_args_listen.conf:2
